maestro-studio/web/src/MockServerInstructions.tsx

```diff
@@ -0,0 +1,35 @@
+import { CodeSnippet } from "./Examples"
+
+const MockServerInstructions = ({ projectId }: { projectId: string }) => (
+  <div className="w-full h-full flex justify-center items-center">
+    <div className="flex flex-col w-3/4 space-y-6 p-8 shadow-xl border-4 border-blue-300 margin-auto">
+      <p className="text-lg font-bold">No events found! Follow the guide below to integrate Maestro SDK.</p>
+      
+      <div>
+        <p className="text-md">First, add the Maestro SDK dependency to your app:</p>
+        <CodeSnippet>{`implementation 'dev.mobile:maestro-sdk-android:1.21.4-SNAPSHOT'`}</CodeSnippet>
```

Comment:
Replace `1.21.4-SNAPSHOT` with `+` **OR** get a version from `gradle.properties`, whichever you prefer